project-resources/community-health-files-guide.md

```diff
@@ -0,0 +1,57 @@
+# Guide to Community Health Files for OpenJS Foundation Projects
+
+Community health files are documents related to project collaboration. Most commonly, this includes:
+
+- `CODE_OF_CONDUCT.md`
+- `CONTRIBUTING.md`
+- `GOVERNANCE.md`
+- `SECURITY.md`
+
+> [!NOTE]
+> See [the full list](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types) of supported default community health files
+
+## Default Community Health Files
+
+Instead of adding these files to every repository within an organization, we can add them only once to a special organization repository which must be named `.github`. GitHub will automatically use the community health files specified at this location by default for all repositories within the organization.
```

Comment:
```suggestion
Instead of adding these files to every repository within a user or organization, we can add them only once to a special user/organization repository which must be named `.github`. GitHub will automatically use the community health files specified at this location by default for all repositories within the user/organization.
```

Reply:
while this is technically correct -- the best kind of correct -- this guide should probably omit this for clarity.  I don't think any openjs projects live under a user account (and if they do, I suggest they should be moved to an org)

Reply:
Fair enough, given the audience here.